### PR TITLE
Add missing step to in-cluster-client-configuration example

### PR DESCRIPTION
**Pull request by dims**
**What this PR does / why we need it**:

If the permissions are not setup correctly the example fails.

Change-Id: I167ef68be66f8b56740236ae475c3b7fdcc0dfb5

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/client-go/examples/in-cluster-client-configuration/README.md [hunk 1]
```diff
@@ -25,7 +25,26 @@ build the image on Minikube:
     docker build -t in-cluster .
 
 If you are not using Minikube, you should build this image and push it to a registry
-that your Kubernetes cluster can pull from.
+that your Kubernetes cluster can pull from. If you have RBAC enabled, use the following
+snippet to create role binding which will grant the default service account view
+permissions.
+
+```
```
Comment (caesarxuchao) [inline staging/src/k8s.io/client-go/examples/in-cluster-client-configuration/README.md]: Could be a one-liner `kubectl create clusterrolebinding default-view --clusterrole=view --serviceaccount=namespace:namespace`, according to [doc](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#default-roles-and-role-bindings).

Comment (caesarxuchao) [reply staging/src/k8s.io/client-go/examples/in-cluster-client-configuration/README.md]: I assume `view` is a default role that exists in every cluster, but couldn't really find documentation to support that.

**Review by timothysc — APPROVED** [review]
/lgtm 
/approve 

@kubernetes/sig-api-machinery-pr-reviews for final approval.

File: staging/src/k8s.io/client-go/examples/in-cluster-client-configuration/README.md (hunk 1 above)
Comment (liggitt) [reply staging/src/k8s.io/client-go/examples/in-cluster-client-configuration/README.md]: yes, it's a default role when RBAC is enabled. see https://kubernetes.io/docs/reference/access-authn-authz/rbac/#default-roles-and-role-bindings

**Pull request merged**